src/users/entities/users.entity.ts

```diff
@@ -0,0 +1,13 @@
+import { User } from '@prisma/client';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UsersEntity implements User {
+  @ApiProperty()
+  id: string;
+
+  @ApiProperty()
+  name: string;
```

Comment:
I think there is no `name` for user in requirements 🤔 There is a `password` though that's missing here

Reply:
I think initial tasks mentions email and name only, password info is present in tasks with register endpoints I think